hello-world/Kbuild

```diff
@@ -3,8 +3,11 @@ helloworld-objs := hello_world.rust.o
 
 CARGO ?= cargo
 
+export c_flags
+export abs_srctree ?= ${CURDIR}
```

Comment:
This is exported by v5.2+ (torvalds/linux@25b146c5b8ceecd43c311552d325a4e403c639f2) but it's pretty easy to keep compatibility with older ones.